accounts/models.py

```
@@ -1,3 +1,53 @@
+    objects = UserManager()
+
+    def __str__(self):
+        return f"{self.username} - {self.email}"
+
+
+class Profile(BaseModel):
+    user = models.OneToOneField('accounts.User', on_delete=models.CASCADE, related_name='profile')
+    birthday = models.DateField(null=True, blank=True)
+    bio = models.TextField()
+    avatar = models.ImageField(upload_to='avatars/', null=True, blank=True)
+    phone_number = models.CharField(max_length=15, null=True, blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)